content/blog/2019/announcing-discuss.istio.io/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Announcing discuss.istio.io
+subtitle: Istio's discussion board
+description: Istio has a new discussion board\.
+publishdate: 2019-01-10
+attribution: The Istio Team
+weight: 100
+---
+
+We in the Istio community have been working to find the right medium for users to engage with other members of the community -- to ask questions,
+to get help from other users, and to engage with developers working on the project.
+
+We’ve tried several different avenues, but each has had some downsides. RocketChat was our most recent endeavor, but the lack of certain
+features (for example, threading) meant it wasn’t ideal for any longer discussions around a single issue. It also led to a dilemma for
+some users -- when should I email istio-users@googlegroups.com and when should I use RocketChat?
+
+We think we’ve found the right balance of features in a single platform, and we’re happy to announce
+[discuss.istio.io](https://discuss.istio.io). It’s a full-featured forum where we will have discussions about Istio from here on out.
+It will allow you to ask a question and get threaded replies! As a real bonus, you can use your GitHub identity.
+
+If you prefer emails, you can configure it to send emails just like Google groups did.
+
+We will be marking our Google groups “read only” so that the content remains, but we ask you to send further questions over to
```

Comment:
what is the guideline for existing emails that still require an answer?  Should we ask users to repost to discuss.istio.io?

Reply:
I added a sentence about needing to repost.